scripts/sanitycheck

```diff
@@ -2043,6 +2138,79 @@ class TestSuite:
                                 lower_better))
         return results
 
+    def testcase_target_report(self, report_file):
+
+        run = "Sanitycheck"
+        eleTestsuite = None
+        append = options.only_failed
+
+        errors = 0
+        passes = 0
+        fails = 0
+        duration = 0
+        skips = 0
+
+        for identifier, ti in self.instances.items():
+            for k in ti.results.keys():
+                if ti.results[k] == 'PASS':
+                    passes += 1
+                elif ti.results[k] == 'BLOCK':
+                    errors += 1
+                elif ti.results[k] == 'SKIP':
+                    skips += 1
+                else:
+                    fails += 1
+
+        eleTestsuites = ET.Element('testsuites')
+        eleTestsuite = ET.SubElement(eleTestsuites, 'testsuite',
+                                     name=run, time="%d" % duration,
+                                     tests="%d" % (errors + passes + fails),
+                                     failures="%d" % fails,
+                                     errors="%d" % errors, skipped="%d" %skips)
+
+        handler_time = "0"
+
+        # print out test results
+        for identifier, ti in self.instances.items():
+            for k in ti.results.keys():
+
+                eleTestcase = ET.SubElement(
+                        eleTestsuite, 'testcase', classname="%s:%s" %(ti.platform.name, os.path.basename(ti.test.name)),
+                        name="%s" % (k), time=handler_time)
+                if ti.results[k] in ['FAIL', 'BLOCK']:
+                    el = None
+
+                    if ti.results[k] == 'FAIL':
+                        el = ET.SubElement(
+                            eleTestcase,
+                            'failure',
+                            type="failure",
+                            message="failed")
+                    elif ti.results[k] == 'BLOCK':
+                        el = ET.SubElement(
+                            eleTestcase,
+                            'error',
+                            type="failure",
+                            message="failed")
+                    p = ("%s/%s/%s" % (options.outdir, ti.platform.name, ti.test.name))
+                    bl = os.path.join(p, "handler.log")
+
+                    if os.path.exists(bl):
+                        with open(bl, "rb") as f:
+                            log = f.read().decode("utf-8")
+                            el.text = log
+
+                elif ti.results[k] == 'SKIP':
+                    el = ET.SubElement(
+                        eleTestcase,
+                        'skipped')
+
+        result = ET.tostring(eleTestsuites)
```

Comment:
There's no CR/LF in the saved file. Is this intentional?

Reply:
just run it through xmllint --format :)
Those files are not intended to be read by humans.